Number Guessing Game/main.py

```diff
@@ -31,7 +31,7 @@ def check_win(player_guess):
     attempts += 1
 
     if check_win(guess):
-        print('You won! - Number of attempts: ' + str(attempts))
+        print('You won! with  - Number of attempts: ' + str(attempts))
```

Comment:
Original message is more straight to the point and clean.
'with' also creates breaks the flow of the sentence, especially when you are putting a hyphen in between to separate the two sentences.

Reply:
okay i understand sir